bundles/org.eclipse.swt/Eclipse SWT/common/org/eclipse/swt/AutoScaling.java

```diff
@@ -0,0 +1,108 @@
+package org.eclipse.swt;
+
+import java.util.*;
+
+import org.eclipse.swt.internal.*;
+
+/**
+ * Utility for checking and enabling monitor-specific auto scaling.
+ * @since 3.132
+ */
```

Comment:
So you want this to be exported as API? If that's the case, make sure that every visible method in it is properly named and documented.